xml/System.Text.Json/JsonSerializer.xml

```diff
@@ -479,6 +487,9 @@ For more information, see [How to serialize and deserialize JSON](~/docs/standar
 
           ]]></format>
         </remarks>
+        <exception cref="ArgumentException"><paramref name="inputType"/> is not compatible with <paramref name="value"/>.</exception>
+        <exception cref="ArgumentNullException"><paramref name="inputType" /> is <see langword="null" />.</exception>
```

Comment:
We forgot to add this `ArgumentNullException` to the source docs: https://github.com/dotnet/runtime/pull/33400/files#diff-0a82308a189816cb9e1159f4b8af905cR35-R41.

Reply:
I added this to a local todo for the next time I touch this file (shouldn't be long)